sql/mozfun/norm/result_type_to_product_name/udf.sql

```diff
@@ -0,0 +1,28 @@
+CREATE OR REPLACE FUNCTION norm.result_type_to_product_name(
+  res STRING
+)
+RETURNS STRING AS (
+    CASE
+    WHEN res IN ('search_suggest', 'search_history', 'search_suggest_rich') THEN 'default_partner_search_suggestion'
+    WHEN res IN ('search_engine') THEN 'search_engine_suggestion'
```

Comment:
From @dzeber 
On a separate note, what about removing suggestions from the name, so changing this to num_search_engine_impressions @rebecca-burwei - this is what we had in the doc but I'm worried it'll be confusing.

From @rebecca-burwei 

yeah, I think num_search_engine_impressions is too confusing. I feel strongly that Product should either choose these names, or be heavily bought in to them. Otherwise, they will just keep asking us what the names mean when it's really their responsibility to define the product experience. So, maybe you could go back to the doc and talk to Tito about it?

Reply:
yup, I've added a comment in the doc and flagged Loren (back next week)